### PR TITLE
Add consistency_level in search/query request

### PR DESCRIPTION
**Pull request by czs007**
issue #24468

### CODE REVIEW

File: internal/proxy/task_search.go [hunk 1]
```diff
@@ -276,10 +276,16 @@ func (t *searchTask) PreExecute(ctx context.Context) error {
 
 	if t.request.GetDslType() == commonpb.DslType_BoolExprV1 {
 		annsField, err := funcutil.GetAttrByKeyFromRepeatedKV(AnnsFieldKey, t.request.GetSearchParams())
-		if err != nil {
-			return errors.New(AnnsFieldKey + " not found in search_params")
+		if err != nil || len(annsField) == 0 {
+			if enableMultipleVectorFields {
```
Comment (longjiquan) [inline internal/proxy/task_search.go:280]: this flag only controls if we support multiple vector fields in the schema, seems not very related with search.

**Review by longjiquan — CHANGES_REQUESTED** [review]

File: internal/proxy/task_search.go (hunk 1 above)
Comment (czs007) [reply internal/proxy/task_search.go:280]: If we enable multi-vector, an error needs to be reported here to prevent missing changes.

**Review by longjiquan — CHANGES_REQUESTED** [review]
/lgtm

**Pull request merged**